Doc/library/sysconfig.rst

```diff
@@ -461,3 +461,31 @@ You can use :mod:`sysconfig` as a script with Python's *-m* option:
 This call will print in the standard output the information returned by
 :func:`get_platform`, :func:`get_python_version`, :func:`get_path` and
 :func:`get_config_vars`.
+
+You can also pass a **--json** flag to have a json-formatted output
+
+.. code-block:: shell-session
+
+    $ python -m sysconfig --json
+      {
+          "platform": "macosx-14.6-arm64",
+          "pyversion": "3.14",
+          "default_scheme": "posix_prefix",
+          "paths": {
+              "stdlib": "/Users/flo/cpython/build/lib/python3.14",
+              "platstdlib": "/Users/flo/cpython/build/lib/python3.14",
+              "purelib": "/Users/flo/cpython/build/lib/python3.14/site-packages",
+              ...
+          },
+          "variables": {
+              "prefix": "/Users/flo/cpython/build",
+              "exec_prefix": "/Users/flo/cpython/build",
+              "py_version": "3.14.0a0",
+              "py_version_short": "3.14",
+              "py_version_nodot": "314",
+              "installed_base": "/Users/flo/cpython/build",
+              "base": "/Users/flo/cpython/build",
+              ...
+          }
+      }
```

Comment:
```suggestion
   $ python -m sysconfig --json
   {
       "platform": "macosx-14.6-arm64",
       "pyversion": "3.14",
       "default_scheme": "posix_prefix",
       "paths": {
           ...
       },
       "variables": {
           ...
       }
   }
```

I aligned your output with the directive (that's what we do) and used 4-space indents for JSON output.